FortnoxAPILibrary/Exceptions/FortnoxTooManyRequestsException.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+
+namespace FortnoxAPILibrary.Exceptions
+{
+    public class FortnoxTooManyRequestsException : Exception
+    {
+        public HttpStatusCode Code { get; set; }
```

Comment:
This field is extra, because we already have clear exception name, which already means 429

Reply:
@anton-beetroot-23071996 Actually Fortnox returned 503 before API update, so I've decidet to left it just in case

Reply:
Resolved